adapters/build.gradle

```diff
@@ -23,7 +23,7 @@ configurations {
 
 dependencies {
     compile 'com.android.support:appcompat-v7:23.3.0'
-
+    compile 'com.android.support:recyclerview-v7:23.3.0'
```

Comment:
`provided` isn't supported for AAR's yet. So we either need to find a work-around for this before releasing it or just document how to avoid including it:

```
compile('io.realm:android-adapters:1.0.1') {
  exclude group: 'com.android.support', module: 'recyclerview-v7'
}
```

In either case I think we should do that outside the scope of this PR